src/utils/loremIpsum.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Utils file to export texts of various lengths to be used in stories and tests.
+ */
+
+export const loremIpsum = `Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quisque vel dolor ac sem congue ornare rhoncus quis tortor.
```

Comment:
we can use a dev dependencies calls https://www.npmjs.com/package/lorem-ipsum to generate dynamically words or sentence

Reply:
a mock should be used to make it work. 
I suggest creating a testsUtils folder for this utility

Reply:
I took a look at the library and it looks like a nice tool. However, it may be a bit too much for the use I actually think we'll make of it in the design system (we could use it for website though 🤔). For the moment, I imported the const inside Typography.mocks.ts, with a comment saying to move it in case it should be used for stories or tests outside Typography